packages/EasyWebhook/docs/webhook.md

```diff
@@ -0,0 +1,129 @@
+---eonx_docs---
+title: 'Webhook class'
+weight: 1000
+---eonx_docs---
+
+# Webhook class
+
+The **Webhook** class defines webhook configuration. Once configured, the [WebhookClient](webhook-client.md) class is
+responsible for sending the webhook.
+
+## Properties
+
+A Webhook object has the following properties:
+
+- `$allowRerun`: Boolean indicating whether a webhook can be rerun multiple times.
+- `$body`: Body of the webhook HTTP request, provided as an array of key-value pairs, which will be formatted as a JSON
```

Comment:
It won't be necessarily formatted to JSON, it depends on the body formatter service used.